### ISSUE TITLE
Error: TypeError: Contract "....." should be marked as abstract. 

### ISSUE THREAD

**Issue by damianlluch** [yusefnapora/lazy-minting#2]
Hi, I am wanting to implement your example, but when I extend my contract with EIP12 I get these errors....
My contract is created with Solidity version 0.8.2. I don't have a constructor

Any suggestions?

![image](https://user-images.githubusercontent.com/1463804/152384983-5854db16-cecf-445a-a763-97c48fbe39aa.png)

**Comment by St4rgarden** [comment]
Yeah so his contract inherits from two other contracts that each have their own constructor.

There's a constructor for both the ERC721 parent contract - and a constructor for the EIP712 contract.  Both of those constructors need arguments for their parameters.

It's a little hard to tell from the information you provided - but it appears you may not have passed the arguments into the constructor for EIP712
